source/module_hamilt_lcao/hamilt_lcaodft/local_orbital_wfc.cpp

```diff
@@ -299,7 +306,15 @@ void Local_Orbital_wfc::wfc_2d_to_grid(
     if (out_wfc_lcao && myid == 0)
     {
         std::stringstream ss;
-        ss << GlobalV::global_out_dir << "LOWF_K_" << ik + 1 << ".dat";
+        if (GlobalV::out_app_flag)
+        {
+            ss << GlobalV::global_out_dir << "LOWF_K_" << ik + 1 << ".dat";
+        }
+        else
+        {
+            ss << GlobalV::global_out_dir << istep << "_"
```

Comment:
Is it better to write the file name as LOWF_K_1_STEP_1? And please add a CI test for this case. i

Reply:
The other matrices are all output in this format. Do I need to change them or maintain them?

Reply:
OK, just keep it.